src/popup/components/AccountInfoCard.vue

```diff
@@ -50,25 +49,25 @@ export default defineComponent({
     BtnPlain,
   },
   props: {
-    account: {
-      type: Object as PropType<IFormSelectOption>,
-      default: () => {},
-    },
-    value: { type: [String, Number], default: null },
+    address: { type: String, required: true },
+    name: { type: String, default: '' },
+    idx: { type: Number, default: 0 },
+    balance: { type: Number, default: 0 },
+    isSelected: Boolean,
+    isAirGapAccount: Boolean,
   },
   setup(props, { root }) {
     const { getAccountBalance } = useBalances({ store: root.$store });
+    const bgColorStyle = computed(() => ({ '--bg-color': getAddressColor(props.address) }));
```

Comment:
In my opinion this is bad to return CSS/HTML code within the JS logic. I suggest to put the object directly into template.